_posts/2024-08-21-render-gml-dictionary-ea-diagram.adoc

```diff
@@ -37,7 +38,61 @@ Each `dictionary_entry` has the following attributes:
 * name
 * description
 
-.Example of using `lutaml_gml_dictionary`
+For example, you want to render the GML Dictionary file named
+`Building_class.xml`.
+
+.GML Dictionary File `Building_class.xml`
+[source,xml]
+```
```

Comment:
@kwkwan AsciiDoc uses 4 dashes `----` for code block delimiters not 3 backticks (that is MarkDown)